src/pages/hotel/booking/booking-confirmation/user-details/UserDetails.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react'
+import { useSelector } from 'react-redux'
+import { IRootState } from '../../../../store/reducers/rootReducer'
 import './UserDetails.css'
 
 function UserDetails() {
+  const firstName = useSelector(
+    (state: IRootState) => state.userDetails.userDetails.firstName,
+  )
+  const lastName = useSelector(
+    (state: IRootState) => state.userDetails.userDetails.lastName,
+  )
+  const email = useSelector(
+    (state: IRootState) => state.userDetails.userDetails.email,
+  )
+  const phone = useSelector(
+    (state: IRootState) => state.userDetails.userDetails.phone,
+  )
```

Comment:
why need each and item. 
 const userDetails = useSelector(
    (state: IRootState) => state.userDetails.userDetails,
  )
 and at the code of diplay just use -> userDetails.firstName etc